### ISSUE TITLE
Can't install nuget package

### ISSUE THREAD

**Issue by sebastianpatten** [maxmind/GeoIP2-dotnet#40]
I can't seem to get the nuget package to install. The Solution is .net 4.5. Any ideas?

```
Added file 'packages.config'.
Successfully added 'Newtonsoft.Json 7.0.1' to newproject.
For adding package 'Newtonsoft.Json 7.0.1' to project 'newproject' that targets 'net45',
>> PowerShell scripts are being executed from 'tools' (not framework-specific)
Executing script file 'C:\Blah\packages\Newtonsoft.Json.7.0.1\tools\install.ps1'.
Adding 'MaxMind.Db 1.2.0' to newproject.
For adding package 'MaxMind.Db 1.2.0' to project 'newproject' that targets 'net45',
>> Assembly references are being added from 'lib\net40'
Added reference 'MaxMind.Db' to project 'newproject'
Added file 'packages.config'.
Successfully added 'MaxMind.Db 1.2.0' to newproject.
Adding 'Microsoft.Bcl.Async 1.0.168' to newproject.
For adding package 'Microsoft.Bcl.Async 1.0.168' to project 'newproject' that targets 'net45',
>> Assembly references are being added from 'lib\net40'
Install failed. Rolling back...
System call failed. (Exception from HRESULT: 0x80010100 (RPC_E_SYS_CALL_FAILED))
```

**Comment by oschwald** [comment]
I haven't seen that one before. It almost sounds like an issue with NuGet. It might be worth following up with them.

**Comment by oschwald** [comment]
Closed by #43.
